<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TSettingsForm</class>
 <widget class="QDialog" name="TSettingsForm">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>267</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MultiCopy - Settings</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/MainIcon_16</normaloff>:/MainIcon_16</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="LanguageLabel">
       <property name="text">
        <string>Language</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="Languages"/>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>(restart required)</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="RAMBuffer">
     <property name="title">
      <string>RAM Buffer</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="CellSize_Label">
        <property name="text">
         <string>Cell size:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="CellSize">
        <property name="suffix">
         <string> MB</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>32</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="CellsCount_Label">
        <property name="text">
         <string>Cells count:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="CellsCount">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>32</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="RequiredRAM_Label">
        <property name="text">
         <string>Required RAM:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="RequiredRAM">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="CopyDateTime">
     <property name="text">
      <string>Copy date and time of folders</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="TotalCalc">
     <property name="text">
      <string>Calculate count and total size of files</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="DirContentsOnly">
     <property name="text">
      <string>Copy folder content only (not same folder)</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="SubDirsDepth_Label">
       <property name="text">
        <string>Depth of copying of subdirectories:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="SubDirsDepth">
       <property name="specialValueText">
        <string>Unlimited</string>
       </property>
       <property name="minimum">
        <number>-1</number>
       </property>
       <property name="maximum">
        <number>10000000</number>
       </property>
       <property name="value">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="OK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>CellSize</tabstop>
  <tabstop>CellsCount</tabstop>
  <tabstop>CopyDateTime</tabstop>
  <tabstop>TotalCalc</tabstop>
  <tabstop>DirContentsOnly</tabstop>
  <tabstop>SubDirsDepth</tabstop>
  <tabstop>OK</tabstop>
  <tabstop>Cancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>CellSize</sender>
   <signal>valueChanged(int)</signal>
   <receiver>TSettingsForm</receiver>
   <slot>calculateRAM()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CellsCount</sender>
   <signal>valueChanged(int)</signal>
   <receiver>TSettingsForm</receiver>
   <slot>calculateRAM()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Cancel</sender>
   <signal>clicked()</signal>
   <receiver>TSettingsForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>376</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>376</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>calculateRAM()</slot>
 </slots>
</ui>
